tests/test_js_graph.py

```diff
@@ -1502,3 +1503,54 @@ def insert_by_parametrized_type(graph, cell_type, tags=None):
         assert False, "Unexpected cell type: {}".format(cell_type)
 
     return cell_id
+
+
+@pytest.mark.parametrize(
+    'layout_name',
+    [
+        QmxGraphApi.LAYOUT_ORGANIC,
+        QmxGraphApi.LAYOUT_COMPACT,
+        QmxGraphApi.LAYOUT_CIRCLE,
+        QmxGraphApi.LAYOUT_COMPACT_TREE,
+        QmxGraphApi.LAYOUT_EDGE_LABEL,
+        QmxGraphApi.LAYOUT_PARALLEL_EDGE,
+        QmxGraphApi.LAYOUT_PARTITION,
+        QmxGraphApi.LAYOUT_RADIAL_TREE,
+        QmxGraphApi.LAYOUT_STACK,
+    ],
+)
+def test_run_all_layouts(layout_name, graph_cases):
+    graph = graph_cases('3v_1e')
+    graph.eval_js_function('api.runLayout', layout_name)
+
+
+def test_run_organic_layout(graph_cases):
+    graph = graph_cases('3v_3e')
+    label = lambda cell: graph.get_label(cell)
+    nodes_positions = {
+        label(v): {
+            'before': None,
+            'after': None,
+        }
+        for v in graph.get_vertices()
+    }
+
+    for v in graph.get_vertices():
+        nodes_positions[label(v)]['before'] = graph.get_vertex_position(v)
+    graph.eval_js_function('api.runLayout', QmxGraphApi.LAYOUT_ORGANIC)
+    for v in graph.get_vertices():
+        nodes_positions[label(v)]['after'] = graph.get_vertex_position(v)
+
+    for position_data in nodes_positions.values():
+        # We do not have the exact expected position to check - But we do know that the positions
+        # should at least change.
+        x_before, y_before = position_data['before']
+        x_after, y_after = position_data['after']
+        assert not pytest.approx(x_before) == x_after and not pytest.approx(y_before) == y_after, \
```

Comment:
Also I guess you could do `assert not pytest.approx(position_data['before']) ==position_data['after'], '...'`.
But you don't need to change that =)